tracer/test/snapshots/Iast.HeaderInjection.AspNetCore5.Vuln.Cookie.SensitiveValue.RedactionEnabled.verified.txt

```diff
@@ -0,0 +1,83 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/headerinjection,
+    Service: Samples.Security.AspNetCore5,
+    Type: web,
+    Tags: {
+      aspnet_core.endpoint: Samples.Security.AspNetCore5.Controllers.IastController.HeaderInjection (Samples.Security.AspNetCore5),
+      aspnet_core.route: iast/headerinjection,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/headerinjection,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/HeaderInjection?useValueFromOriginHeader=False,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "HEADER_INJECTION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "Set-Cookie: "
+          },
+          {
+            "redacted": true,
+            "pattern": "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqrs",
```

Comment:
Where is this value coming from, I can't seem to find it? 🤔

Reply:
This is a redaction pattern that is replacing the real value